acmw-db-app/styles/components/CompanyForm.module.css

```diff
@@ -0,0 +1,37 @@
+.popup {  
+    position: fixed;  
+    width: 100%;  
+    height: 100%;  
+    top: 0;  
+    left: 0;  
+    right: 0;  
+    bottom: 0;  
+    margin: auto;  
+    background-color: rgba(0,0,0, 0.5);  
+  }  
+  .popup_inner {  
+    position: absolute;  
```

Comment:
I think we should probably avoid using absolute positioning in this case because resizing the window also resizes the popup and it could make the inside things behave strangely. But also I'm not sure what the best way to deal with this is since idk if we want to make the inside stuff scale down too. Maybe we can make it scroll if it gets too small? Sounds complicated haha

Reply:
Oh good to know! I don't know much about CSS so I can see if there's some better way to implement this.

Reply:
Oh wait a minute, I think I totally misinterpreted sorry! I just realized it's for the position of the popup not the stuff inside of it. I think it's fine then to have it centered. I'm wondering if making it a flex container will help make all things inside things adjust to its changing size. I think instead of having styles `one` and `two`, we can make a "Search By" div section using flex display w/wrap so the text inputs will either be next to either if there's room, or they will be on separate lines if it gets that small.
I don't know that's just an idea. I'm not all that good with css either. But basically I think absolute is actually okay.